core/src/main/scala/org/apache/spark/api/python/StreamingPythonRunner.scala

```diff
@@ -91,6 +91,11 @@ private[spark] class StreamingPythonRunner(
       new BufferedInputStream(pythonWorker.get.channel.socket().getInputStream, bufferSize))
 
     val resFromPython = dataIn.readInt()
+    if (resFromPython != 0) {
+      val errMessage = PythonWorkerUtils.readUTF(dataIn)
+      throw new RuntimeException(s"Runner initialization failed (returned $resFromPython). " +
```

Comment:
Should be a PythonException? (comparing it with similar code in `StreamingForeachBatchHelper.scala`).
Also, add 'Streaming' at the start of the message.